### PR TITLE
Create 2019-01-07--js-core-dev-team-weekly.md

### PR DESCRIPTION
**Pull request by jacobheun**

### CODE REVIEW

File: meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    - 0.34 release now blocked on an [issue handling control signals](https://ci.ipfs.team/blue/organizations/jenkins/IPFS%2Fjs-ipfs/detail/PR-1803/3/pipeline/257) (also in master 😱)
+        - I am on holiday this week - would appreciate someone looking into this!
+- Next:
+    - n/a - on holiday
+
+@vasco-santos
+- Done:
+  - IPNS spec ready, reviewed and merged [ipfs/specs#184](https://github.com/ipfs/specs/pull/184)
+  - IPNS / DHT interop reviews by Alan were resolved [ipfs/interop#36](https://github.com/ipfs/interop/pull/36), [ipfs/interop#39](https://github.com/ipfs/interop/pull/39) and [ipfs/interop#47](https://github.com/ipfs/interop/pull/47)
+  - JS libp2p-daemon-client with basic commands and dht almost ready [libp2p/js-libp2p-daemon-client#1](https://github.com/libp2p/js-libp2p-daemon-client/pull/1)
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md]: Fantastic! One thing that was unclear to me was https://github.com/libp2p/js-libp2p-daemon-client/pull/1/files#r245892243

If it is a real client (accessing remotely), which RPC library are you using?

File: meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+- Blocked:
+- Next:
+  - Prep for libp2p team week (all next week)
+  - Fix libp2p-crypto interface for [libp2p-crypto#129](https://github.com/libp2p/js-libp2p-crypto/issues/129)
+  - Finish initial daemon work PR [libp2p-daemon#1](https://github.com/libp2p/js-libp2p-daemon/pull/1)
+
+@hugomrdias
+- Done:
+  - Holidays and catching up
+  - Bubbling up bundle size PRs
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md]: @hugomrdias where is the tracking issue for this?

File: meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  - Met with @alanshaw, @mcollina, @vasco-santos and @jacobheun to discuss switch/multistream/mplex. 
+- Blocked:
+- Next:
+  - Merge remaining bundle size PRs
+  - Add bundle size cmd to aegir
+  - Finish Error codes proposal
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md]: @hugomrdias where is the wip proposal?

Comment (alanshaw) [reply meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md]: https://github.com/ipfs/js-ipfs/pull/1746

File: meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md (hunk 1 above)
Comment (vasco-santos) [reply meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md]: This uses unix sockets for communication between processes executing on the same host (through protobuf)

More information in the [spec](https://github.com/libp2p/go-libp2p-daemon/blob/master/specs/CONTROL.md#technical-details) and [roadmap](https://github.com/libp2p/go-libp2p-daemon/blob/1d7d0a25b6152441e4e8a216d53ee2f7240a161e/ROADMAP.md#introduction)

File: meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md (hunk 3 above)
Comment (jacobheun) [inline meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md]:
```suggestion
  - Finish Error codes proposal [js-ipfs#1746](https://github.com/ipfs/js-ipfs/pull/1746)
```

File: meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md (hunk 2 above)
Comment (hugomrdias) [reply meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md]: there isn't an issue, i have a PR open in all the repos

Comment (alanshaw) [reply meeting-notes/2019/Q1/2019-01-07--js-core-dev-team-weekly.md]: Can we get a link to the sheet plz?

**Pull request merged**